com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -397,11 +414,22 @@ internal Vector3 Scale
         /// <param name="dirtyTime"></param>
         protected void TryCommitTransformToServer(Transform transformToCommit, double dirtyTime)
         {
+            TryCommitTransform(transformToCommit, dirtyTime);
+        }
+
+
+        protected void TryCommitTransform(Transform transformToCommit, double dirtyTime)
```

Comment:
This is the same function signature as TryCommitTransformToServer, and that one just calls this one... who not just rename that one?